assets/javascripts/lib/apply-reply.js.es6

```diff
@@ -0,0 +1,36 @@
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default function(replyId, replyTitle, replyContent, model) {
+  // Replace variables with values.
+  if (model) {
+    const vars = {
+      user: model.get("user.username"),
+      original_poster: model.get("topic.posters")
+        ? model.get("topic.posters")[0].user.get("username")
+        : "",
+      reply_to: model.get("post.reply_to_user.username"),
+      last_user: model.get("topic.last_poster_username"),
+      reply_to_or_last:
+        model.get("post.reply_to_user.username") ||
+        model.get("topic.last_poster_username")
+    };
+
+    for (var key in vars) {
+      if (vars[key]) {
+        replyTitle = replyTitle.replace("#{" + key + "}", vars[key]);
```

Comment:
Let's use `%{variable_name}` instead of `#{variable_name}` so that it's the same syntax as translated text.